packages/calcite-components/src/components/tabs/tabs.e2e.ts

```diff
@@ -405,5 +405,24 @@ describe("calcite-tabs", () => {
 
       expect(selectedTitleOnEmit).toBe("Tab 2 Title");
     });
+
+    it("should hide x when last closable tab and display x when > 1 closable tabs", async () => {
```

Comment:
For extra coverage, could you update or add a test for closing tabs 1–3? If adding a new test, could you group them in a `describe` block?